.github/workflows/test_rust.yml

```diff
@@ -65,9 +84,14 @@ jobs:
       # Additionally: https://lld.llvm.org/ELF/start-stop-gc
       - name: Disable linker start-stop-gc
         # Note: We also use `rust-lld` on Windows, see `config.toml`.
-        if: runner.os != 'macOS'
+        if: runner.os == 'Linux'
```

Comment:
Doesn't that comment need some adjustment? While it's still true, it's now relevant somewhat differently now... As in, the interesting point is not that we also have to do this on Windows because `rust-lld` is also used there, but _not_ having to do it _despite_ that...? :thinking:

Reply:
removed the comment altogether as it seems it's an ELF issue